FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt/common/main.c

```diff
@@ -26,7 +26,7 @@
  */
 
 /***
- * See https://www.FreeRTOS.org/mqtt/index.html for configuration and usage instructions.
+ * See https://www.FreeRTOS.org/mqtt_lts/index.html for configuration and usage instructions.
```

Comment:
There are a few places  we would need to update this .url file, coupl[e of readmes I think. Can you please update those as well?

Reply:
Updated